edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore

```diff
@@ -0,0 +1,82 @@
+                _.each( context.providers, function( provider) {
+                    if ( provider.registerUrl ) { %>
+                        <button type="button" class="button button-primary button-<%- provider.id %> login-provider register-<%- provider.id %>" data-provider-url="<%- provider.registerUrl %>">
+                            <div class="icon <% if ( provider.iconClass ) { %>fa <%- provider.iconClass %><% } %>" aria-hidden="true">
+                                <% if ( provider.iconImage ) { %>
+                                    <img class="icon-image" src="<%- provider.iconImage %>" alt="<%- provider.name %> icon" />
+                                <% } %>
+                            </div>
+                            <span aria-hidden="true"><%- provider.name %></span>
+                            <span class="sr"><%- _.sprintf( gettext("Create account using %(providerName)s."), {providerName: provider.name} ) %></span>
+                        </button>
+                <%  }
+                }); %>
+
+                <% if ( context.hasSecondaryProviders ) { %>
+                    <button type="button" class="button-secondary-login form-toggle" data-type="institution_login">
+                        <%- gettext("Use my institution/campus credentials") %>
+                    </button>
+                <% } %>
+            </div>
+            <div class="section-title lines">
+                <h3>
+                    <span class="text"><%- gettext("or create a new one here") %></span>
+                </h3>
+            </div>
+        <% } else { %>
+            <h2><%- gettext('Create an account')%></h2>
+        <% } %>
+    <% } else if (context.autoRegisterWelcomeMessage) { %>
+        <span class="auto-register-message"><%- context.autoRegisterWelcomeMessage %></span>
+    <% } %>
+
+    <%= context.fields /* xss-lint: disable=underscore-not-escaped */ %>
+
+    <div class="form-field checkbox-optional_fields_toggle">
+        <input type="checkbox" id="toggle_optional_fields" class="input-block checkbox"">
+        <label for="toggle_optional_fields">
+            <span class="label-text">
+                <%- gettext("Support education research by providing additional information") %>
+            </span>
+        </label>
+    </div>
+
+
+    <button type="submit" class="action action-primary action-update js-register register-button">
+    	<% if ( context.registerFormSubmitButtonText ) { %><%- context.registerFormSubmitButtonText %><% } else { %><%- gettext("Create account") %><% } %>
+    </button>
+</form>
+
+<script type="text/javascript">
+	// This removes all capital letters, except the first one for the label-text class
+	for(label of document.getElementsByClassName('label-text')) {
+		let content = label.textContent.toLocaleLowerCase();
+		content = content.charAt(0).toUpperCase() + content.slice(1);
+		label.textContent = content;
+	}
+
+	var xpath = "//a[text()='Terms of Service and Honor Code']";
```

Comment:
What is this for? @luismorenolopera

Reply:
is the "Terms of Service and Honor Code" link selector, to change for "Terms of Service"

Reply:
Is there any other way to do this? Can you point me to the file where this 'Terms of Service and Honor Code' appears? @luismorenolopera

Reply:
https://github.com/Pearson-Advance/edx-platform/blob/pearson-release/juniper.stage/openedx/core/djangoapps/user_authn/views/registration_form.py#L960, changing it here would generate a change in the other sites